CDCgov/prime-simplereport
# Boban/5619 fix upload errors

# FRONTEND PULL REQUEST

## Related Issue

- Resolves #5619 

## Changes Proposed

- Remove extra characters like `"` and `$`from error messages
- Fix bug for incorrect row number when columns are missing

## Additional Information

- Column missing error is fixed by checking for that specific exception and it's not the last row in the file.

## Testing

- Ensure the errors messages no longer have any extra characters
- Make sure row number is correct for errors
